MAINTAINERS.md

```diff
@@ -6,31 +6,23 @@ This is the list of maintainers, including their email addresses for direct comm
 
 | Name                  | GitHub Id                                            | email                           | Area of expertise                                                             |
 |-----------------------|------------------------------------------------------|---------------------------------|-------------------------------------------------------------------------------|
-| Makoto Takemiya       | [@takemiyamakoto](https://github.com/takemiyamakoto) | takemiya@soramitsu.co.jp        | Product vision                                                                |
-| Dmitri Venger         | [@dmitrivenger](https://github.com/dmitrivenger)     | venger@soramitsu.co.jp          | Project manager                                                               |
-| Bogdan Mingela        | [@Mingela](https://github.com/Mingela)               | mingela@soramitsu.co.jp         | Iroha Team Lead                                                               |
-| Aleksandr Petrosyan   | [@appetrosyan](https://github.com/appetrosyan)       | ap886@cantab.ac.uk              | Iroha 2 architect                                                             |
-| Daniil Polyakov       | [@Arjentix](https://github.com/Arjentix)             | daniil.polyakov@soramitsu.co.jp | Development: Rust, C++                                                        |
+| Makoto Takemiya       | [@takemiyamakoto](https://github.com/takemiyamakoto) | takemiya@soramitsu.co.jp        | Product Owner                                                                |
+| Dmitri Venger         | [@dmitrivenger](https://github.com/dmitrivenger)     | venger@soramitsu.co.jp          | Project manager/Team lead                                                               |
+| Bogdan Mingela        | [@Mingela](https://github.com/Mingela)               | mingela@soramitsu.co.jp         | Iroha tribe Lead                                                               |
```

Comment:
Since it's called 'Area of expertise' I'd change that to smth more clear, e.g. Product Manager or I can be the Team Lead, leaving you Project Manager etc